apply/apply.go

```diff
@@ -60,6 +60,8 @@ func initializeEngine(dsn, source, driverName, path string, options ...morph.Eng
 		return nil, err
 	}
 
+	options = append(options, morph.LockDB("mutex_migration"))
```

Comment:
Should we expose the lock key as a flag with `mutex_migration` as default? Might be problematic if you use the library and then try via the CLI again.